lib/global.js

```diff
@@ -1,14 +1,3 @@
-var createTorrent = require('create-torrent')
-var wrtc = require('wrtc')
+var wrtc = require('electron-webrtc')()
 
 global.WRTC = wrtc
-
-global.WEBTORRENT_ANNOUNCE = createTorrent.announceList
```

Comment:
Sorry if I wasn't clear enough in my comment. I'd like to keep automatically adding wss:// trackers to all torrents in `webtorrent-hybrid` for the time being. Eventually, once folks start adding wss:// trackers to .torrent files, we can remove it.

If we remove it now, then torrents downloaded with `webtorrent-hybrid` will not be seeded to web peers by default.